jekyll/_cci2/examples-intro.md

```diff
@@ -30,12 +30,12 @@ jobs:
           name: Update npm
           command: 'sudo npm install -g npm@latest'
       - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
+          key: dependency-cache-{{ checksum "package-lock.json" }}
```

Comment:
So, technically, this example cannot use `package-lock` because it's < npm v5.x.

Can you either bump up the version of the node/mongo versions to match [this document](https://circleci.com/docs/2.0/language-javascript/) or revert back to `package.json` (hopefully the former).

Reply:
Will revert this change. Maybe you can open a seperate hacktoberfest issue for this.